src/mmu/paging.c

```diff
@@ -23,6 +23,24 @@ page_table_t* next_table_create(page_table_t* table, uint64_t index);
 page_table_t* get_p4();
 void paging_set_entry(page_entry_t* entry, uint64_t addr, uint64_t flags);
 
+uint32_t paging_flags_for_entry(page_entry_t* entry) {
```

Comment:
I'd rather declare the prototype of this function below the other ones (L25) and add its definition at the bottom of the file.

Reply:
Done